tst/EnergyPlus/unit/Coils/CoilCoolingDXFixture.hh

```diff
@@ -49,14 +49,14 @@
 #include <gtest/gtest.h>
 
 // EnergyPlus Headers
-#include "../Fixtures/EnergyPlusFixture.hh"
+#include "../Fixtures/SQLiteFixture.hh"
 #include <EnergyPlus/Data/EnergyPlusData.hh>
 
-class CoilCoolingDXTest : public EnergyPlus::EnergyPlusFixture {
+class CoilCoolingDXTest : public EnergyPlus::SQLiteFixture {
 public:
 protected:
   void SetUp() override {
-    EnergyPlus::EnergyPlusFixture::SetUp(); // Sets up the base fixture first.
+    EnergyPlus::SQLiteFixture::SetUp(); // Sets up the base fixture first.
```

Comment:
Gotcha, just made this inherit the sqlite fixture so that you can test the sql output.